employees/models.py

```diff
@@ -79,13 +79,17 @@ def clean(self) -> None:
 
         if hasattr(self, "author"):
             if not isinstance(self.work_hours, timedelta):
-                raise ValidationError(message=ReportValidationStrings.WORK_HOURS_FIELD_NOT_TIMEDELTA_INSTANCE.value)
+                raise ValidationError(message=ReportValidationStrings.WORK_HOURS_WRONG_FORMAT.value)
+            if self.work_hours < ReportModelConstants.MIN_WORK_HOURS.value:
+                raise ValidationError(message=ReportValidationStrings.WORK_HOURS_MIN_VALUE_NOT_EXCEEDED.value)
+            if self.work_hours.seconds % timedelta(minutes=15).seconds != 0:
```

Comment:
`seconds` should work with our cases but using `total_seconds()` is always safer.

Reply:
You're right. Done :)